recipes/pcaudiolib/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -euo pipefail
+
+./autogen.sh
+# custom LD for windows needed
+LD=$LD ./configure --prefix=$(realpath $PREFIX)
+make prefix=$(realpath $PREFIX)
+make install prefix=$(realpath $PREFIX)
```

Comment:
We don't allow shipping static libraries. Please remove libpcaudio.a as the last build step.

Reply:
Could I do this by `--disable-static` instead?

Reply:
You can if that is an option recognized by the build script.